### PR TITLE
Task/24 - Clients

### PR DESCRIPTION
**Pull request by jordanrafalm**

### CODE REVIEW

File: app/src/main/res/layout/fragment_clients.xml [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="360dp"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_clients.xml:5]: match_parent

File: app/src/main/res/layout/fragment_clients.xml [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+    android:layout_height="263dp"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_clients.xml:6]: match_parent

File: app/src/main/res/layout/fragment_clients.xml [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <ImageView
+        android:id="@+id/image_adsystem"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="208dp"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_clients.xml:31]: do not hardcode margins, in this task you could delete most of them, take a look at:
https://medium.com/@nomanr/constraintlayout-chains-4f3b58ea15bb

**Review by przemek19980102 — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/dimens.xml [hunk 4]
```diff
@@ -14,6 +14,42 @@
     <dimen name="margin_6x">48dp</dimen>
     <dimen name="margin_6.5x">52dp</dimen>
     <dimen name="margin_7x">56dp</dimen>
+    <dimen name="margin_7.5x">60dp</dimen>
```
Comment (przemek19980102) [inline app/src/main/res/values/dimens.xml:17]: woah... please remove all those, we don't need that many, we aren't probably using even half of that!

File: app/src/main/res/layout/fragment_clients.xml [hunk 5]
```diff
@@ -0,0 +1,181 @@
...
+        app:layout_constraintHorizontal_chainStyle="spread"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_chainStyle="spread" />
+
+    <ImageView
+        android:id="@+id/image_airgap"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/margin_37x"
```
Comment (przemek19980102) [inline app/src/main/res/layout/fragment_clients.xml:47]: Try to implement this layout using as few margins as possible, this is way too many

**Review by przemek19980102 — CHANGES_REQUESTED** [review]

**Review by msarelo — CHANGES_REQUESTED** [review]
- W PR powinny być tylko Twoje zmiany.
- Spróbuj zaimportować pliki SVG zamiast PNG
- Marginesy powinny dopasowywane względem elementów - tak jak wyjaśniłem ostatnio na spotkaniu.

**Pull request merged**